azurerm/resource_arm_storage_share_test.go

```diff
@@ -49,15 +56,51 @@ func TestAccAzureRMStorageShare_disappears(t *testing.T) {
 			{
 				Config: config,
 				Check: resource.ComposeTestCheckFunc(
-					testCheckAzureRMStorageShareExists("azurerm_storage_share.test", &sS),
-					testAccARMStorageShareDisappears("azurerm_storage_share.test", &sS),
+					testCheckAzureRMStorageShareExists(resourceName, &sS),
+					testAccARMStorageShareDisappears(resourceName, &sS),
 				),
 				ExpectNonEmptyPlan: true,
 			},
 		},
 	})
 }
 
+func TestAccAzureRMStorageShare_updateQuota(t *testing.T) {
+	var sS storage.Share
+
+	ri := acctest.RandInt()
+	rs := strings.ToLower(acctest.RandString(11))
+	config := testAccAzureRMStorageShare_basic(ri, rs, testLocation())
+	config2 := testAccAzureRMStorageShare_updateQuota(ri, rs, testLocation())
+	resourceName := "azurerm_storage_share.test"
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testCheckAzureRMStorageShareDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: config,
+				Check: resource.ComposeTestCheckFunc(
+					testCheckAzureRMStorageShareExists(resourceName, &sS),
+				),
+			},
+			{
+				Config: config2,
+				Check: resource.ComposeTestCheckFunc(
+					testCheckAzureRMStorageShareExists(resourceName, &sS),
+					resource.TestCheckResourceAttr(resourceName, "quota", "5"),
+				),
+			},
+			{
+				ResourceName:      resourceName,
+				ImportState:       true,
+				ImportStateVerify: true,
```

Comment:
we can remove the Imports from this test (since they're covered by the Basic test above; in general we don't Import test updates since they're covered elsewhere)

Reply:
Done